src/Sylius/Bundle/AdminBundle/Resources/views/Security/Login/Content/Form/_submitButton.html.twig

```diff
@@ -0,0 +1,3 @@
+{% import '@SyliusAdmin/Shared/Helper/button.html.twig' as _button %}
+
+{{ _button.primary({ text: 'sylius.ui.login'|trans, type: 'submit', class: 'btn btn-primary w-100' }) }}
```

Comment:
I use `_` to indicate it's something imported. Same as I later started to use `_context.<var>` instead of just `<var>` to indicate from where something is coming. IMO it helps to understand from where some value is taken.